### DISCUSSION TITLE
编译无法通过，toolchain-aarch64_cortex-a53_gcc-13.2.0_musl 错误

### DISCUSSION THREAD

**Discussion by liyiranlab** [immortalwrt/immortalwrt#1241] [Q&A]
### Describe the bug

make[8]: *** [Makefile:1929: readelf.o] Error 1
make[8]: Leaving directory '/home/ran/AX6/immortalwrt/build_dir/toolchain-aarch64_cortex-a53_gcc-13.2.0_musl/binutils-2.40/binutils'
make[7]: *** [Makefile:1437: all-recursive] Error 1
make[7]: Leaving directory '/home/ran/AX6/immortalwrt/build_dir/toolchain-aarch64_cortex-a53_gcc-13.2.0_musl/binutils-2.40/binutils'
make[6]: *** [Makefile:881: all] Error 2
make[6]: Leaving directory '/home/ran/AX6/immortalwrt/build_dir/toolchain-aarch64_cortex-a53_gcc-13.2.0_musl/binutils-2.40/binutils'
make[5]: *** [Makefile:4062: all-binutils] Error 2
make[5]: Leaving directory '/home/ran/AX6/immortalwrt/build_dir/toolchain-aarch64_cortex-a53_gcc-13.2.0_musl/binutils-2.40'
make[4]: *** [Makefile:1005: all] Error 2
make[4]: Leaving directory '/home/ran/AX6/immortalwrt/build_dir/toolchain-aarch64_cortex-a53_gcc-13.2.0_musl/binutils-2.40'
make[3]: *** [Makefile:116: /home/ran/AX6/immortalwrt/build_dir/toolchain-aarch64_cortex-a53_gcc-13.2.0_musl/binutils-2.40/.built] Error 2
make[3]: Leaving directory '/home/ran/AX6/immortalwrt/toolchain/binutils'
time: toolchain/binutils/compile#1.33#0.44#2.17
    ERROR: toolchain/binutils failed to build.
make[2]: *** [toolchain/Makefile:93: toolchain/binutils/compile] Error 1
make[2]: Leaving directory '/home/ran/AX6/immortalwrt'
make[1]: *** [toolchain/Makefile:89: /home/ran/AX6/immortalwrt/staging_dir/toolchain-aarch64_cortex-a53_gcc-13.2.0_musl/stamp/.toolchain_compile] Error 2
make[1]: Leaving directory '/home/ran/AX6/immortalwrt'
make: *** [/home/ran/AX6/immortalwrt/include/toplevel.mk:233: world] Error 2

### ImmortalWrt version

mater

### ImmortalWrt release

23.05

### ImmortalWrt target/subtarget

86eb3e4

### Device

redmi AX6

### Image kind

Self-built image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by science2468** [comment]
别编译了。用image builder。https://firmware-selector.immortalwrt.org/

**Comment by xlighting2017** [comment]
我这里是好的
<img width="897" alt="image" src="https://github.com/immortalwrt/immortalwrt/assets/25970065/e4b69b52-3bd9-4ad2-8317-78f79f18bc3f">

**Comment by liyiranlab** [comment]
喜欢自己编译自己需要的软件。

**Reply by liyiranlab** [reply]
我都是本地自己编译的。

**Comment by liyiranlab** [comment]
从新拉库编译通过了
